.github/workflows/release.yaml

```diff
@@ -0,0 +1,81 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.*'
+
+jobs:
+  build:
+    name: Build Binaries
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        goos: [linux, windows, darwin]
+        goarch: [amd64, arm64]
+        wasm-tools-version: ["1.216.0"]
```

Comment:
Wondering if we could move this dependency to a config file (.env? cargo.yaml?) since it’s used in multiple workflow jobs.

Reply:
There is a GitHub repo variable which we can use to store wasm-tools-version: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#creating-configuration-variables-for-a-repository

Reply:
both .env and cargo.yaml need to be parsed by github action. I am not opposing to it though.

Reply:
I was able to move this to an `env:` section at the top level in test.yaml.

Reply:
removed wasm-tools